src/PKSim.Infrastructure/ORM/Repositories/ValueOriginRepository.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System.Collections.Generic;
+using OSPSuite.Core.Domain;
+using OSPSuite.Utility.Collections;
+using OSPSuite.Utility.Extensions;
+using PKSim.Core.Repositories;
+using PKSim.Infrastructure.ORM.Mappers;
+
+namespace PKSim.Infrastructure.ORM.Repositories
+{
+   public class ValueOriginRepository : IValueOriginRepository
+   {
+      private readonly IFlatValueOriginRepository _flatValueOriginRepository;
+      private readonly IFlatValueOriginToValueOriginMapper _valueOriginMapper;
+
+      private static readonly ValueOrigin _defaultValueOrigin = new ValueOrigin
```

Comment:
This is our default here. Not sure if the value for Default should be true or false by default :)

Reply:
False is probably ok